modules/core/src/Build.js

```diff
@@ -88,5 +88,44 @@ export default class Build extends Core {
         },
       }),
     )
+
+    await this.buildServiceWorker()
+  }
+
+  async buildServiceWorker() {
```

Comment:
Is this really something that can be inside concept? Could you use Concepts after method?

Reply:
I removed manifest file generation from there, and moved it to server route generation, however I had to keep SW generation there, because of need to look at `dist` directory for built files (which are then precached in the `sw.js` file).